tutorials/tts/Inference_ModelSelect.ipynb

```diff
@@ -230,9 +231,6 @@
     "            \"mixerttsx\": \"tts_en_lj_hifigan_ft_mixerttsx\"\n",
     "        }\n",
     "        pretrained_model = spectrogram_generator2ft_hifigan.get(spectrogram_generator, \"tts_hifigan\")\n",
-    "    elif audio_generator == \"univnet\":\n",
```

Comment:
@ericharper, could you return these three lines? Main has them.

Reply:
Tried to revert the changes. Can you check if it's correct?